docs/configuration/template_reference.md

```diff
@@ -74,6 +74,7 @@ versions.
 | reReplaceAll  | pattern, replacement, text | string | [Regexp.ReplaceAllString](https://golang.org/pkg/regexp/#Regexp.ReplaceAllString) Regexp substitution, unanchored. |
 | graphLink  | expr | string | Returns path to graph view in the [expression browser](https://prometheus.io/docs/visualization/browser/) for the expression. |
 | tableLink  | expr | string | Returns path to tabular ("Table") view in the [expression browser](https://prometheus.io/docs/visualization/browser/) for the expression. |
+| parseDuration | string | string | Parses a duration string such as "1h" into the integer number of seconds it represents. |
```

Comment:
Why not using a float? model.duration supports milliseconds.

Reply:
I have thought a bit more: we could return a float and then let the user use `printf`.